.dlc.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
What's the purpose of this file?

Reply:
This file I would like to check dead link in doc

Reply:
What about we introduce this file after we enable the dead link check in the future? I'm not sure links in this file currently are related to Flink CDC's doc.